erpnext/buying/doctype/supplier/supplier.json

```diff
@@ -97,7 +97,7 @@
   {
    "fieldname": "default_bank_account",
    "fieldtype": "Link",
-   "label": "Default Bank Account",
+   "label": "Default Company Bank Account",
```

Comment:
Default Bank Account seems appropriate too. A party is not necessarily bound to a company. Also unnecessarily long label

Reply:
Based on the same review https://github.com/frappe/erpnext/pull/20632#pullrequestreview-359493500 this is the same label as for the "default bank account" field in the customer doctype, so it would be more consistent if both fields have the same label:
https://github.com/frappe/erpnext/blob/3c25308b27458413dafaf3094247ec2b1f326d95/erpnext/selling/doctype/customer/customer.json#L125-L130